src/views/customer/CustomerListView/Toolbar.tsx

```diff
@@ -41,6 +41,8 @@ const Toolbar: React.FC<any> = ({ className, ...rest }) => {
             <Box maxWidth={500}>
               <TextField
                 fullWidth
+                value={ inputSearch }
+                onChange={ onChange }
```

Comment:
inputSearchChange

Reply:
The name `onChange` is too generic. Please come up with more specific name.
Since you have `value= { searchCustomer }` you can use `onChange={ searchCustomerChange }`

Reply:
Tomorow I will send new commit, also with update customer details.